### PR TITLE
adding event_timestamp for camus partitioning

### LINKED ISSUE #8: Add event_timestamp to event
For Camus to be able to drop the event in the right partition, it needs an `event_timestamp` field.

@johnjmartin

### PR DESCRIPTION
**Pull request by johnjmartin**
Closes #8 
After closing this I will bump the version on Cookbooks

### CODE REVIEW

File: src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java [hunk 1]
```diff
@@ -77,11 +79,21 @@ public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
...
 
         producer.send(new ProducerRecord<>(TOPIC_NAME, queryCompletedEvent.getMetadata().getQueryId(), queryEventJson.toString()));
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " text `" + queryCompletedEvent.getMetadata().getQuery() + "`");
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " cpu time (minutes): " + queryCompletedEvent.getStatistics().getCpuTime().getSeconds()/60 + " wall time (minutes): " + queryCompletedEvent.getStatistics().getWallTime().getSeconds()/60.0);
     }
+
+    public static String getCurrentTimeStamp()
+    {
+        long yourmilliseconds = System.currentTimeMillis();
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```
Comment (drdee) [inline src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java]: You can move this one to the constructor.

File: src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java [hunk 2]
```diff
@@ -77,11 +79,21 @@ public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
...
         log.debug("Sending " + queryEventJson.toString() + " to Kafka Topic: " + TOPIC_NAME);
 
         producer.send(new ProducerRecord<>(TOPIC_NAME, queryCompletedEvent.getMetadata().getQueryId(), queryEventJson.toString()));
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " text `" + queryCompletedEvent.getMetadata().getQuery() + "`");
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " cpu time (minutes): " + queryCompletedEvent.getStatistics().getCpuTime().getSeconds()/60 + " wall time (minutes): " + queryCompletedEvent.getStatistics().getWallTime().getSeconds()/60.0);
     }
+
+    public static String getCurrentTimeStamp()
+    {
+        long yourmilliseconds = System.currentTimeMillis();
```
Comment (drdee) [inline src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java]: why not call it `currentTimeMillis`?

File: src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java [hunk 3]
```diff
@@ -77,11 +79,21 @@ public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
...
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+        Date resultdate = new Date(yourmilliseconds);
+        String strDate = sdf.format(resultdate);
```
Comment (drdee) [inline src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java]: directly return?

File: src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java [hunk 4]
```diff
@@ -77,11 +79,20 @@ public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
...
 
         producer.send(new ProducerRecord<>(TOPIC_NAME, queryCompletedEvent.getMetadata().getQueryId(), queryEventJson.toString()));
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " text `" + queryCompletedEvent.getMetadata().getQuery() + "`");
         log.debug("QID " + queryCompletedEvent.getMetadata().getQueryId() + " cpu time (minutes): " + queryCompletedEvent.getStatistics().getCpuTime().getSeconds()/60 + " wall time (minutes): " + queryCompletedEvent.getStatistics().getWallTime().getSeconds()/60.0);
     }
+
+    public static String getCurrentTimeStamp(String datePattern)
+    {
+        long currentTimeMilliseconds = System.currentTimeMillis();
+        SimpleDateFormat sdf = new SimpleDateFormat(datePattern);
```
Comment (drdee) [inline src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java]: I meant to move the `SimpleDataFormat` to the constructor so you don't create this object every time you emit the event.

Comment (johnjmartin) [inline src/main/java/com/shopify/presto/eventlisteners/QueryLogEventListener.java]: Oh okay, that totally makes more sense now

**Pull request merged**